js/canvas.js

```diff
@@ -299,9 +312,13 @@ func_buttons.forEach(button=>{
 
         if(button['title']==='Eraser/Color Eraser' && button['id']==='btn-pressed'){
 
+            cursorEraser.style.display = "block";
+
             document.addEventListener('keydown', (e)=>{
                 if(e.keyCode===107){++width;}
                 if(width!==3 && e.keyCode===109){--width;}
+                cursorEraser.style.width = width + "px";
```

Comment:
Could you do one small favor by hiding the eraser square when the cursor goes out of the canvas?

Reply:
Yeah sure

Reply:
Just update the PR with commit c505ed4

Reply:
Sure thing!!! Thanks a ton!!